doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1031,23 +934,26 @@ The central server backs up
 the database (excluding the schema_migrations table and the database schema) and
 the directories `/etc/xroad/` and `/etc/nginx/sites-enabled/`.
 
+Backups contain sensitive information that must be kept secret (for example, private keys and database credentials). In other words, leaking this information could easily lead to full compromise of central server. Therefore, it is highly recommended that backup archives are encrypted and stored securely. Should the information still leak for whatever reason the central server should be considered as compromised and reinstalled from scratch.
+
+Central server backups are signed and optionally encrypted. The GNU Privacy Guard [GnuPG] is used for encryption and signing. Central server's backup encryption key is generated during central server initialisation. In addition to the automatically generated backup encryption key, additional public keys can be used to encrypt backups.
+
 ## 12.1 Backing Up the System Configuration
 
 To back up the configuration, follow these steps.
-1. On the Management menu, select Back Up and Restore.
-2. Click Back Up Configuration to start the backup process.
-3. A window opens displaying the output from the backup script; click OK to close it.
-4. When done, the configuration backup file appears in the list of configuration backup files.
+1. On the tab Settings, select Back Up and Restore sub-tab.
+2. Click Back up config. to start the backup process.
+3. When done, the configuration backup file appears in the list of configuration backup files.
 
 ## 12.2 Restoring the System Configuration in the User Interface
 
 To restore configuration, follow these steps.
-1. On the Management menu, select Back Up and Restore.
+1. On the tab Settings, select Back Up and Restore sub-tab.
 2. Select a file from the list of configuration backup files and click Restore.
-3. Click Confirm to proceed.
-4. A window opens displaying the output from the restore script; click OK to close it.
+3. Confirm to proceed.
+4. A popup notification shows after the restore whether the restoring was successful or not.
 
-If something goes wrong while restoring the configuration it is possible to revert back to the old configuration. Central Server stores so called pre-restore configuration automatically to `/var/lib/xroad/conf_prerestore_backup.tar`. Either move it to `/var/lib/xroad/backup/` folder and utilize the user interface to restore it or use the command line interaface described in the next chapter.
+If something goes wrong while restoring the configuration it is possible to revert back to the old configuration. Central Server stores so called pre-restore configuration automatically to `/var/lib/xroad/conf_prerestore_backup.tar`. Either move it to `/var/lib/xroad/backup/` folder and utilize the user interface to restore it or use the command line interface described in the next chapter.
```

Comment:
Since the pre-restore configuration is a `.tar` file, it probably cannot be restored using the UI since the UI is expecting `.gpg` files. Command line should still be OK, but using some specific switches with the restore command is required.

> Either move it to `/var/lib/xroad/backup/` folder and utilize the user interface to restore it or use the command line interface described in the next chapter.